styles/objects/_layout.scss

```diff
@@ -0,0 +1,10 @@
+.obj-layout {
+  margin-top: space("lg");
+  margin-left: space("md");
+  margin-right: space("md");
+
+  @media (min-width: 40rem) {
+    margin-left: space("2xl");
+    margin-right: space("2xl");
+  }
```

Comment:
What I like to do for flexible containers is set a maximum width and use auto margins to center it. That gets rid of a media query, ensures a consistent amount of spacing, and stops growing on super large screens.

```css
  // subtracting 3rem give us 1.5rem on either side of the container
  width: calc(100% - space("2xl"));
  max-width: 72rem;
  margin-left: auto;
  margin-right: auto;
```